server/src/test/java/io/crate/planner/operators/JoinTest.java

```diff
@@ -986,6 +986,40 @@ public void test_nested_joins_with_non_existing_columns_raises_error() throws Ex
 
     }
 
+    @Test
+    public void test_using_clause_specifying_multiple_columns_with_a_non_existing_column_raises_error() throws IOException {
+        var executor = SQLExecutor.builder(clusterService, 2, Randomness.get(), List.of())
+            .addTable("CREATE TABLE j1 (x INT, y INT)")
+            .addTable("CREATE TABLE j2 (x INT)")
+            .build();
+
+        assertThatThrownBy(() -> executor.analyze(
+            """
```

Comment:
super super minor, since the queries are very short, you could just use normal `"..."` strings and make them one line.